deploy_monitoring.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- name: deploy control plane monitoring
+  hosts: control-plane
+  connection: local
+  gather_facts: false
+  vars:
+    setup_yaml:
+      - url: https://raw.githubusercontent.com/metal-stack/releases/{{ metal_stack_release_version }}/release.yaml
+        meta_var: metal_stack_release
```

Comment:
Not related to this PR but we should move this variable to `group_vars/all/images.yaml` and remove it from the rest of the playbooks.